OpenKeychain/src/test/java/org/sufficientlysecure/keychain/pgp/PgpEncryptDecryptTest.java

```diff
@@ -464,7 +464,7 @@ public void testAsymmetricSignDetached() {
     @Test
     public void testAsymmetricEncryptDecrypt() {
 
-        String plaintext = "dies ist ein plaintext ☭" + TestingUtils.genPassphrase(true);
+        String plaintext = "dies ist ein plaintext ☭" + TestingUtils.testPassphrase0;
```

Comment:
I don't think appending passphrases to plaintext like this is necessary, just keep the plaintext on its own

Reply:
Yeah, I know this is pretty silly, but I'll rather do a seperate PR for cleaning up the tests a bit.

Reply:
Ok, resolved.